api/python/quilt3/packages.py

```diff
@@ -619,6 +619,10 @@ def download_manifest(dst):
 
             pkg = cls._from_path(local_pkg_manifest)
             pkg._origin = PackageRevInfo(str(registry.base), name, top_hash)
+
+            if pkg.top_hash != top_hash:
```

Comment:
OK so doing this check in `_browse` also gets `install` (in PR title)? And works for subpackage install as well? Since the user can install any subpackage and we have the manifest at that point I think we only need to check the installed object hashes and not the tophash.